### PR TITLE
Bora branch

### PR DESCRIPTION
**Pull request by BoraPuka**

### CODE REVIEW

File: assets/movie-look.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      font-size: 9px !important;
+    }
+      .labels {
+      position: absolute;
+      top: 1rem !important;
+      right: 1rem !important;
+    }
+  }
+  
+  @media (min-width: 790px) and (max-width: 1920px) {
```
Comment (Shero-Inc) [inline assets/movie-look.css]: Hello @BoraPuka, we use the 768px as a breakpoint. Can you check it, please?

File: assets/movie-look.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    .movie-container > * {
+      gap: 7rem;
+    }
+  }
+  
+  @media (max-width: 719px) {
```
Comment (Shero-Inc) [inline assets/movie-look.css]: Please change the breakpoint also here.

File: assets/movie-look.css [hunk 3]
```diff
@@ -163,13 +163,13 @@
     }
   }
   
-  @media (min-width: 790px) and (max-width: 1920px) {
+  @media (min-width: 768px) {
```
Comment (dejvid-sherocommerce) [inline assets/movie-look.css:166]: test comment

Comment (dejvid-sherocommerce) [reply assets/movie-look.css:166]: tet

**Review by Shero-Inc — APPROVED** [review]

**Pull request merged**